client/src/components/BurgerButton.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+export const StyledBurger = styled.button`
+  top: 5%;
+  left: 2rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  width: 2rem;
+  height: 2rem;
+  background: transparent;
+  border: none;
+  cursor: pointer;
+  padding: 0;
+  z-index: 10;
+
+  &:focus {
+    outline: none;
+  }
+
+  div {
+    width: 2rem;
+    height: 0.25rem;
+    background: black;
+    border-radius: 10px;
+    transition: all 0.3s linear;
+    position: relative;
+    transform-origin: 1px;
+
+    :first-of-type {
+      transform: ${props => (props.open ? 'rotate(45deg)' : 'rotate(0)')};
```

Comment:
Isn't there an easier way to set css if the component is open?
`rotate(0)`, `opacity: 1` ,.. doesn't look required.

Does something like this work?

```
     ${props =>props.open ? `
      :first-of-type {
        transform: rotate(45deg);
      }

      :nth-of-type(2) {
      opacity: 1;
      transform: translateX(20px);
    }
     `: ''};
```

Or make use of https://emotion.sh/docs/composition

```
const open = css`
      :first-of-type {
        transform: rotate(45deg);
      }

      :nth-of-type(2) {
      opacity: 1;
      transform: translateX(20px);
    }
`
```

```
  ${props => props.open ? open : ''}
```

Reply:
Changed this one to emotion composition now. Works fine. 
`opacity: 1` changed to `opacity: 0` so middle `<div />` fades away on click.